src/Data/Yaml.php

```diff
@@ -24,22 +24,8 @@ class Yaml extends Handler
 	 */
 	public static function encode($data): string
 	{
-		// TODO: The locale magic should no longer be
```

Comment:
@lukasbestle i think you added this code originally? If so, could you double check that it's ok that we remove it now with PHP 8?

Reply:
We have a unit test for this that still passes. 👍